index.bs

```diff
@@ -1709,14 +1706,15 @@ and/or <code>URL.{{URL/createFor()}}</code>:
   Append the string "blob"
   (that is, the Unicode code point sequence U+0062, U+006C, U+006F, U+0062)
   to |result|.
-2. Append the ":" (U+003A COLON) character to |result|.
-3. Let |O| be the Blob URL's [=url/origin=].
-  If the <a>Unicode Serialization of an Origin algorithm</a> on |O| returns null,
-  user agents may substitute an implementation defined value for the return value of the <a>Unicode Serialization of an Origin algorithm</a>.
-  Append the result of the <a>Unicode Serialization of an Origin algorithm</a> for |O| [[!RFC6454]] to |result|.
-4. Append the "/" character (U+0024 SOLIDUS) to |result|.
-5. Generate a UUID [[RFC4122]] as a Unicode string and append it to |result|.
-6. Return |result|.
+1. Append the ":" (U+003A COLON) character to |result|.
+1. Let |settings| be the [=incumbent settings object=]
+1. Let |origin| be |settings|'s [=environment settings object/origin=].
+1. Let |serialized| be the <a lt="ASCII serialization of an origin">ASCII serialization</a> of |origin|.
+1. If |serialized| is "null", set it to an implementation-defined value.
+1. Append |serialized| to |result|.
+1. Append the "/" character (U+0024 SOLIDUS) to |result|.
+1. Generate a UUID [[RFC4122]] as a Unicode string and append it to |result|.
+1. Return |result|.
```

Comment:
https://github.com/w3c/web-platform-tests/pull/4657 has some tests for at least the origin serialization part (and at least firefox and chrome both ascii serialize the origin, as expected). Haven't actually added tests to check the rest of the format of these urls.